git_machete/client.py

```diff
@@ -1746,7 +1746,9 @@ def __derive_remote_and_github_org_and_repo(self) -> Tuple[str, Tuple[str, str]]
         if not org_and_repo_for_github_remote:
             raise MacheteException(
                 fmt('Remotes are defined for this repository, but none of them '
-                    'corresponds to GitHub (see `git remote -v` for details)'))
+                    'corresponds to GitHub (see `git remote -v` for details)\n'
+                    f'Actual remotes from git:\n {self.__git.popen_git("remote", "-v")}\n'
+                    f'Actual remotes from git-machete: {self.__git.remotes()}'))
```

Comment:
This line is maybe too low-level to display 😅 also, I think with the addition of `self.__remotes_cached = None` in `flush_caches` the error is no longer likely to happen right?

Reply:
right, i'll just remove those extended messages.